microsoft-edge/extensions-chromium/getting-started/part1-simple-extension.md

```diff
@@ -16,7 +16,13 @@ The goal for this tutorial is to build a Microsoft Edge extension, starting with
 *   Adding icons.
 *   Opening a default pop-up dialog.
 
-If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1). The source code has been updated from Manifest V2 to Manifest V3.
+If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1).  The source code uses Manifest V3.
```

Comment:
```suggestion
If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1).
```
i doubt this article's sentences should mention MV2 or MV3 at all.  The demo code already states "mv3".

Reply:
Extensions repo part 1 still says MV2:
https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part1/part1/manifest.json
`"manifest_version": 2,`

Extensions repo Part 2 already says MV3:
https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/manifest.json
`"manifest_version": 3,`

Reply:
removed statement about whether Extensions repo part 1 uses MV2 or MV3.